src/test/java/uk/gov/hmcts/ccd/BaseTest.java

```diff
@@ -126,9 +126,13 @@
     @Mock
     protected SecurityContext securityContext;
 
+    @Inject
+    private CacheManager cacheManager;
+
     @Before
     @BeforeEach
     public void initMock() throws IOException {
+        cacheManager.getCacheNames().stream().forEach(cacheName -> cacheManager.getCache(cacheName).clear());
```

Comment:
Can't we disable cache using spring boot properties in tests?

Reply:
Good point, I had got so intent on getting everything to work as is with minimal changes - but yes that makes perfect sense